etc/cvmfs/default.conf

```diff
@@ -4,3 +4,9 @@ if [ "$(($$ % 2))" == 1 ]; then
 else
   CVMFS_PAC_URLS="http://grid-wpad/wpad.dat;http://wpad/wpad.dat;http://cernvm-wpad.cern.ch/wpad.dat;http://cernvm-wpad.fnal.gov/wpad.dat"
 fi
+
+# Automatically detect DNS nameserver changes
+CVMFS_DNS_ROAMING=on
```

Comment:
Is this something we should be setting for EGI & OSG too?

Reply:
Maybe.  It is particularly useful for roaming laptops.  It comes with the small overhead of installing a watch point on /etc/resolv.conf.  I guess grid worker nodes would not change the nameserver but the setting should also not hurt.

Reply:
I made prs #31, #32, & #33 to add this to the other 3 branches.

Reply:
Oh now I see that this already was enabled by default in /etc/cvmfs/default.conf, so why do we want it also in the config repo's default.conf?

Reply:
Yep, that's unnecessary. Removed.